float32 x
float32 y
float32 z
string label
float32 last_detected_time
